controllers/main.go

```diff
@@ -44,6 +45,8 @@ const (
 	// This is an artificial limit and can be increased by adjusting the
 	// ticket/fee address indexes above 10000.
 	MaxUsers = 10000
+	// agendasCacheLife is the amount of time to keep agenda data in memory.
+	agendasCacheLife = time.Hour * 24
```

Comment:
24 hours may be a little excessive, I was thinking more like 1 hour would suffice